app/webpacker/components/Tickets/TicketWorkbenches/CompetitionResultActionerView/MiscActions/EventsMergedData/index.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+import { Button, Modal, Popup } from 'semantic-ui-react';
+import EventsMergedDataContent from './EventsMergedDataContent';
+
+export default function EventsMergedData({ ticketDetails }) {
+  const [isModalOpen, setIsModalOpen] = useState();
+
+  return (
+    <>
+      <Popup
+        trigger={(
+          <Button onClick={() => setIsModalOpen(true)}>
+            Events Merged Data
+          </Button>
+      )}
```

Comment:
The indentation on this line looks off. Might just be the GH viewer, though

Reply:
Yes it was off. No idea why eslint formatter didn't fix this. Fixed it now.